pkg/promclient/promclient.go

```diff
@@ -164,7 +166,7 @@ func (c *Client) ExternalLabels(ctx context.Context, base *url.URL) (labels.Labe
 	span, ctx := tracing.StartSpan(ctx, "/prom_config HTTP[client]")
 	defer span.Finish()
 
-	body, _, err := c.req2xx(ctx, &u, http.MethodGet)
+	body, _, err := c.req2xx(ctx, &u, http.MethodGet, "")
```

Comment:
```suggestion
	body, _, err := c.req2xx(ctx, &u, http.MethodGet, EMPTY_HEADER)
```
You can define a constant for empty tenant header, would be easy to read and understand what does `""` means.
Feel free to name variable other than `EMPTY_HEADER` :)